.vscode/cesium-omniverse-windows.code-workspace

```diff
@@ -427,7 +427,10 @@
     "python.analysis.typeCheckingMode": "basic",
     "python.linting.enabled": true,
     "python.linting.pylintEnabled": false,
-    "python.linting.flake8Enabled": true
+    "python.linting.flake8Enabled": true,
+    "[python]": {
+      "editor.defaultFormatter": "ms-python.black-formatter"
```

Comment:
`ms-python.black-formatter` should be added to the extension recommendations list below

Reply:
Good call. Added.